### DISCUSSION TITLE
(FlowFuse) Email alerts for devices and a suggestion for an internal email node

### DISCUSSION THREAD

**Discussion by GogoVega** [FlowFuse/flowfuse#3310] [Ideas]
Hi Team,

Would it be possible to add an "Alerts" section to devices in the "Settings"?
In order to be notified if:
- NR has crashed
- NR is in Safe Mode
- Device not responding
- Device restarted manually (suggestion)

Can I also suggest creating a node like Project nodes allowing to send an email alert with content and a desired team/user?

Let's imagine that a device monitors an electric motor, this device could send an alert to the maintenance team that there is abnormal wear. This node would be an internal version of the existing [email](https://flows.nodered.org/node/node-red-node-email) node (I mean that instead of defining an email we define a FF team or a FF user).

Thank you for your feedback

**Comment by hardillb** [comment]
We added alerts to instances recently, extending to devices is probably the next step. The instance alerts use existing audit log events as triggers and device level audit logs will be in an upcoming release. Once we include sending audit events for things like device crash we can look at hooking up the email alerts.

Would need to think about the email nodes, it is a different usecase, but I'm not sure if there will be enough demand for it

**Reply by GogoVega** [reply]
Thanks,

Let's say that it allows to automate a surveillance system.
A example use that I would see is:
- Every day the maintenance manager receives an email with a summary of the machines status
- As soon as a (planned) intervention occurs, the maintenance team receives an email with status
- As soon as there is a problem (not planned), the maintenance manager receives an email with the status
... I can continue but I think you understand the principle.

This should be highlighted because this is what industries are looking for, in any case this is how Industry 4.0 is taught.

**Reply by GogoVega** [reply]
Hi @hardillb, please take a look at https://github.com/FlowFuse/flowfuse/issues/579
